rclcpp/include/rclcpp/timer.hpp

```diff
@@ -112,8 +112,10 @@ class GenericTimer : public TimerBase
    * \param[in] period The interval at which the timer fires.
```

Comment:
missing parameter doc

Reply:
Added in 04621963e960f136f058d3430ed485ade457e5c6.